dev3/2024-07-02/example-metadata/rembi-modules.json

```diff
@@ -0,0 +1,115 @@
+{
+    "@context": [
+        "https://w3id.org/ro/crate/1.1/context",
+        {
+            "organism_classification": "https://schema.org/taxonomicRange",
+            "BioChemEntity": "https://schema.org/BioChemEntity",
+            "channel": "https://w3id.org/reproduceme#Channel",
+            "obo": "http://purl.obolibrary.org/obo/",
+            "FBcv": "http://ontobee.org/ontology/FBcv",
+            "acquisiton_method": {
+                "@reverse": "https://schema.org/result",
+                "@type": "@id"
+            },
+            "biological_entity": "https://schema.org/about",
+            "biosample": "http://purl.obolibrary.org/obo/OBI_0002648",
+            "preparation_method": "https://www.wikidata.org/wiki/Property:P1537",
+            "specimen": "http://purl.obolibrary.org/obo/HSO_0000308"            
+        }
+    ],
+    "@graph": [
+        {
+            "@type": "CreativeWork",
+            "@id": "ro-crate-metadata.json",
+            "conformsTo": {
+                "@id": "https://w3id.org/ro/crate/1.1"
+            },
+            "about": {
+                "@id": "./"
+            }
+        },
+        {
+            "@id": "./",
+            "@type": "Dataset",
+            "name": "Light microscopy photo of a fly",
+            "description": "Light microscopy photo of a fruit fly.",
+            "licence": "https://creativecommons.org/licenses/by/4.0/",
+            "hasPart": {
+                "@id": "./dros-mel-image.zarr/"
+            }
+        },
+        {
+            "@id": "./dros-mel-image.zarr/",
+            "@type": "Dataset",
+            "name": "OME-ZARR files",
+            "description": "the ome zarr files of the fly.",
+            "biosample": [
+                "_b:01"
+            ]
+        },
+        {
+            "@id": "_:b01",
+            "@type": [
+                "biosample"
+            ],
+            "biological_entity": [
+                "_:b2"
+            ]
+        },
+        {
+            "@id": "_:b02",
+            "@type": [
+                "FBcv:0003045"
```

Comment:
If in the context you add '/' to the end of the FBcv uri, this end up pointing to https://ontobee.org/ontology/FBcv/0003405, but i suspect it should point to: https://ontobee.org/ontology/FBcv_0003405 (though that doesn't appear to exist on ontobee, even if it is referenced in FBcv documents), so i'm not sure what's happened there. I recommend looking the n-quads view in the json-ld playground to double check URIs make sense.

Reply:
I updated the ref to FBcv_0003405